### ISSUE TITLE
Distribute to hockeyapp suddenly failing

### ISSUE THREAD

**Issue by MarkMolina** [KrauseFx/fastlane#400]
In version 1.6.0 I could successfully upload to hockeyapp by using the following settings.

```
ipa(
      workspace: "APPNAME.xcworkspace",
      configuration: "Beta",
      scheme: "Beta",
      destination: "build",
      identity: "Company BV",
      clean: true,
    )

hockey(
      api_token: "OUR_TOKEN",
      ipa: './build/APPNAME.ipa',
      notes_type: "1",
      notes: changelog,
    )
```

After upgrade (1.7.0) I get to following error:

```
Error when trying to upload ipa to HockeyApp: {"status"=>"error", "message"=>"App could not be created."}
```

Didn't touch anything and the app still lives in HockeyApp. Only thing that I changed is the ruby version (2.2.2).

**EDIT**
Tried to upload with the curl command from hockeyapp docs:
```
curl \
> -F "status=2" \
> -F "notes=Manual curl upload" \
> -F "notes_type=0" \
> -F "ipa=@APPNAME.ipa" \
> -F "dsym=@APPNAME.app.dSYM.zip" \
> -H "X-HockeyAppToken: API_TOKEN" \
> https://rink.hockeyapp.net/api/2/apps/APP_TOKEN/app_versions/upload
```

upload seems to work but can't install. Error: Unable to Download App.

**Comment by KrauseFx** [comment]
So the upload work but you can't install it on your device? That sounds a lot like a code signing issue.

**Comment by MarkMolina** [comment]
Well... It only works if I use the curl command from terminal. Upload doesn't work with the hockeyapp command from the fastfile.

**Comment by BobObiPayleven** [comment]
it happened to me just delete the app and recreate it on hockey and you are done

**Comment by clooth** [comment]
I had this problem as well, but it went away when I gave the `hockey` action the application's `public_identifier` parameter.

**Comment by jjepsen** [comment]
I had the same error message and public_identifier worked for me too. Thanks!

**Comment by KrauseFx** [comment]
Great to hear :+1:

**Comment by soffes** [comment]
Same here. `public_identifier` is required.

**Comment by micnguyen** [comment]
Just so this doesn't happen: https://xkcd.com/979/ for when someone comes along and sees this: My Hockey uploads were working fine and then suddenly stopped working. The reason is that I changed my app's bundle ID, which means the referenced HockeyApp app no longer is associated, so it was trying to upload it to an App ID that doesn't exist. I fixed it by editing my HockeyApp app to match my new bundle ID of my actual app.
